ui/src/app/views/workspace-list/component.css

```diff
@@ -112,4 +106,64 @@
   width: 32px;
   height: 32px;
   color: #216FB4;
-}
\ No newline at end of file
+}
+
+.header-area {
```

Comment:
More question than comment. Did you see the same issues on these cards as we saw on notebook and cohort card headers? If so, I just want to add a comment to that ticket, noting it's also present here.